instrument/Lagrange_Definition.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version='1.0' encoding='ASCII'?>
+<instrument xmlns="http://www.mantidproject.org/IDF/1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Lagrange" valid-from="1901-01-01 00:00:00" valid-to="2100-01-31 23:59:59" last-modified="2023-02-06 14:34:26.766171" xsi:schemaLocation="http://www.mantidproject.org/IDF/1.0 http://schema.mantidproject.org/IDF/1.0/IDFSchema.xsd">
+  <!--
+       This is the instrument definition file of the 
+       Generated file, PLEASE DO NOT EDIT THIS FILE!
+       This file was automatically generated by mantidgeometry/ILL/IDF/salsa_generateIDF.py
+
+       For more information, please visit
+       https://www.ill.eu/instruments-support/instruments-groups/instruments/salsa/characteristics/
+       -->
```

Comment:
I see that you had the inspiration from Salsa :)

Reply:
Yes indeed, shamelessly copied methods from both SALSA and D11.